sdk/python/kfp/dsl/_metadata.py

```diff
@@ -152,3 +152,34 @@ def _annotation_to_typemeta(annotation):
   else:
     return TypeMeta()
   return arg_type
+
+def _create_pipeline_metadata_from_pipeline_func(func):
```

Comment:
Please add some comments for the function instead of using long function names.
Could we change the name to _extract_pipelinemetada() for example?

Reply:
> Could we change the name to _extract_pipelinemetada() for example?

Thanks for concrete and actionable proposal. I'll rename the function.